acm-operator/uninstall.sh

```diff
@@ -41,6 +41,7 @@ oc get service | grep "multicluster" | awk '{ print $1 }' | xargs oc delete serv
 
 oc get scc | grep "multicluster" | awk '{ print $1 }' | xargs oc delete scc --wait=false --ignore-not-found
 oc get scc | grep "multicloud" | awk '{ print $1 }' | xargs oc delete scc --wait=false --ignore-not-found
+oc get scc | grep "kui-proxy" | awk '{ print $1 }' | xargs oc delete scc --wait=false --ignore-not-found
```

Comment:
Let's not add these changes to the uninstall as discussed via slack!  Thanks for the PR though!

Reply:
Fixed in latest commit